hudi-common/src/main/java/org/apache/hudi/common/bootstrap/index/DefaultBootstrapIndex.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.hudi.common.bootstrap.index;
+
+import org.apache.hudi.common.model.BootstrapFileMapping;
+import org.apache.hudi.common.table.HoodieTableMetaClient;
+
+import java.util.*;
+
+public class DefaultBootstrapIndex extends BootstrapIndex {
```

Comment:
can you please add some java docs as to why we have this bootstrap index. 
@Xoln : Also, can you help me understand even w/ this new bootstrap index, users needs to specify the bootstrap index class config right? (hoodie.bootstrap.index.class). Why can't we make this class default value for this config. So, even if users don't set them, we will default to this. Guess that was your intention if I am not wrong.

Reply:
yes , you are right.  When init AbstractTableFileSystemView , hoodie.bootstrap.index.class read the default config and to be init . I think it's better to be the index which has no operation and has not any dependencies. (Because there is not check whether index is null at some code)